src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -31,6 +31,7 @@ categories :
       verbosity   : MPI_T_VERBOSITY_USER_BASIC
       scope       : MPI_T_SCOPE_LOCAL
       description : >-
+        Capability Sets:
```

Comment:
What's the intention for adding this tag? IMO it doesn't improve the `README.envvar` content, but I might be missing a use case.